### PR TITLE
Revamp the UI of the Login & Registration page

### PR DESCRIPTION
**Pull request by savindi7**
### Purpose

desktop view
Screenshot 2023-10-23 at 18 22 56

mobile view
Screenshot 2023-10-23 at 18 23 28

### Related Issues
- https://github.com/wso2/product-is/issues/17100

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/private-key-jwt/pages/private-key-jwt-config.tsx [hunk 1]
```diff
@@ -60,19 +59,19 @@ export const PrivateKeyJWTConfig: FunctionComponent<PrivateKeyJWTConfigPageInter
...
+        ? (
+            <Section
+                data-componentid={ `${componentId}-settings-section` }
+                description={ t("console:manage.features.jwtPrivateKeyConfiguration.description") }
+                icon={ getSettingsSectionIcons().jwtPrivateKeyValidation }
+                header={ t(
+                    "console:manage.features.jwtPrivateKeyConfiguration.pageTitle" 
+                ) }
+                onPrimaryActionClick={ handleSelection }
+                primaryAction={ "Configure" }
```
Comment (pavinduLakshan) [inline apps/console/src/features/private-key-jwt/pages/private-key-jwt-config.tsx:72]: Shall we i18n this string :)

File: apps/console/src/features/server-configurations/pages/connector-listing-page.tsx [hunk 2]
```diff
@@ -26,18 +26,20 @@ import {
...
-import { GridLayout, PageLayout } from "@wso2is/react-components";
+import { PageLayout } from "@wso2is/react-components";
 import { AxiosError } from "axios";
+import camelCase from "lodash-es/camelCase";
 import React, { FunctionComponent, MutableRefObject, ReactElement, useEffect, useRef, useState } from "react";
 import { useTranslation } from "react-i18next";
 import { useDispatch, useSelector } from "react-redux";
 import { Dispatch } from "redux";
-import { Divider, Grid, Placeholder, Ref } from "semantic-ui-react";
+import { Divider, Placeholder, Ref } from "semantic-ui-react";
```
Comment (pavinduLakshan) [inline apps/console/src/features/server-configurations/pages/connector-listing-page.tsx:42]: Maybe we can use Divider from oxygen ui as well?

File: apps/console/src/features/server-configurations/pages/connector-listing-page.tsx [hunk 3]
```diff
@@ -272,6 +282,77 @@ export const ConnectorListingPage: FunctionComponent<ConnectorListingPageInterfa
...
     };
 
+    /**
+     * Render connector categories which generate the connector forms dynamically.
+     */
+    const renderDynamicCategoryConnectors = (): ReactElement[] => {
+        if (
+            connectorCategories &&
+          Array.isArray(connectorCategories) &&
+          connectorCategories.length > 0
```
Comment (pavinduLakshan) [inline apps/console/src/features/server-configurations/pages/connector-listing-page.tsx:292]:
```suggestion
          connectorCategories &&
          Array.isArray(connectorCategories) &&
          connectorCategories.length > 0
```

**Review by pavinduLakshan — APPROVED** [review]

File: apps/console/src/features/server-configurations/pages/connector-listing-page.tsx (hunk 3 above)
Comment (savindi7) [reply apps/console/src/features/server-configurations/pages/connector-listing-page.tsx:292]: This gives a lint error

File: apps/console/src/features/server-configurations/pages/connector-listing-page.tsx (hunk 2 above)
Comment (savindi7) [reply apps/console/src/features/server-configurations/pages/connector-listing-page.tsx]: removed the divider

File: apps/console/src/features/private-key-jwt/pages/private-key-jwt-config.tsx (hunk 1 above)
Comment (savindi7) [reply apps/console/src/features/private-key-jwt/pages/private-key-jwt-config.tsx]: fixed https://github.com/wso2/identity-apps/pull/4297/commits/547ecfea29e748359a3c27617f2eccfa4a6d2418

File: apps/console/src/features/server-configurations/pages/connector-listing-page.tsx [hunk 4]
```diff
@@ -189,25 +191,26 @@ export const ConnectorListingPage: FunctionComponent<ConnectorListingPageInterfa
...
+        for (let loadedPlaceholders: number = 0; loadedPlaceholders <= 1; loadedPlaceholders++) {
+            placeholders.push(
+                <Grid xs={ 12 } lg={ 6 } key={ loadedPlaceholders }>
                     <div 
                         className="ui card fluid settings-card"
                         data-testid={ `${testId}-loading-card` }
                     >
                         <div className="content no-padding">
-                            <div className="header-section">
+                            <div className="header-section" style={ { "paddingBottom": "50px" } }>
```
Comment (Achintha444) [inline apps/console/src/features/server-configurations/pages/connector-listing-page.tsx:207]: shall we move this inline css to a seperate file ?

Comment (savindi7) [reply apps/console/src/features/server-configurations/pages/connector-listing-page.tsx]: fixed https://github.com/wso2/identity-apps/pull/4297/commits/e20e1633f26f50be611842ccc3f5b9cc3b49b142

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**